router/xgress_edge/factory_test.go

```diff
@@ -107,7 +108,7 @@ func Test_load(t *testing.T) {
 const (
 	connectMaxQueuedConnections   = 50
 	connectMaxOutstandingConnects = 100
-	connectTimeoutMs              = 3000
```

Comment:
Did you intend to change the value from 3s to 1s?

Reply:
Yes, this is what I commented on below. I don't know how it was passing before as the value it is compared against was 1000. I looked through the commit history and it has always been 1000 since the creation of the file 3 years ago... very strange.